recipes/mrcpp/meta.yaml

```diff
@@ -0,0 +1,109 @@
+
+{% if mpi != "nompi" %}
+{% set mpi_prefix = "mpi_" + mpi %}
+{% else %}
+{% set mpi_prefix = "nompi" %}
+{% endif %}
+# add build string so packages can depend on
+# mpi or nompi variants
+# dependencies:
+# `mrcpp * mpi_mpich_*` for mpich
+# `mrcpp * mpi_*` for any mpi
+# `mrcpp * nompi_*` for no mpi
+
+# mpi builds require the right mpi
+# non-mpi builds *do not* appear to require non-mpi builds
+# at least not always
+
+{% if mpi != 'nompi' %}
+{% set build_pin = mpi_prefix + '_*' %}
+{% else %}
+{% set build_pin = '' %}
+{% endif %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MRChemSoft/mrcpp/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # Add "-undefined dynamic_lookup" linker flag on osx
+    - patches/osx_undefined-dynamic-lookup.patch  # [osx]
+    # Add "override" keyword to silence warnings
+    - patches/inconsistent-missing-override.patch
+    # Instantiate "build_grid" explicitly for "Gaussian"
+    - patches/instantiate-build-grid-for-gaussian.patch
+
+build:
+  number: {{ build }}
+  skip: true  # [win]
+  binary_relocation: true
+  string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}"
+  run_exports:
+    - {{ pin_subpackage('mrcpp', max_pin='x.x') }}
+  run_exports:
+    - {{ name }} * {{ mpi_prefix }}_*
+
+requirements:
+  build:
+    - cmake >=3.11
+    - {{ compiler('cxx') }}
+    - {{ pin_compatible('eigen', max_pin='x.x') }}
+    - ninja
+  host:
+    - eigen
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - {{ mpi }}    # [mpi != 'nompi']
+  run:
+    - eigen
+
+test:
+  commands:
+    # Verify library
+    - test -f $PREFIX/lib/libmrcpp$SHLIB_EXT                              # [not win]
+    - test -L $PREFIX/lib/libmrcpp$SHLIB_EXT                              # [not win]
+    # Inspect linkage
+    - ldd -v $PREFIX/lib/libmrcpp$SHLIB_EXT                               # [linux]
+    - otool -L $PREFIX/lib/libmrcpp$SHLIB_EXT                             # [osx]
+    - conda inspect linkages --show-files --groupby=dependency $PKG_NAME  # [not win]
+    - conda inspect objects -p $PREFIX $PKG_NAME                          # [osx]
+
+outputs:
+  - name: mrcpp
+    files:
+      - bin/mrcpp-tests        # [not win]
+      - include/MRCPP          # [not win]
+      - lib/libmrcpp*          # [not win]
+      - share/cmake/MRCPP      # [not win]
+      - share/MRCPP/mwfilters  # [not win]
+
+about:
+  home: https://github.com/MRChemSoft/mrcpp
+  dev_url: https://github.com/MRChemSoft/mrcpp
+  doc_url: https://mrcpp.readthedocs.io/en/latest/
+  doc_source_url: https://github.com/MRChemSoft/mrcpp/blob/master/doc/index.rst
+  license: LGPL-3.0-or-later
+  license_url: https://opensource.org/licenses/LGPL-3.0
+  license_file: LICENSE
+  license_family: LGPL
+  summary: "MultiResolution Computational Program Package"
+
+extra:
+  recipe-maintainers:
+    - robertodr
+    - bjorgve
```

Comment:
@bjorgve are you willing to be a maintainer?

Reply:
@scopatz hi yes I am.